ci/prow/perf_ops_config.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// data definitions that are used for the config file generation of performance
+// tests cluster maintanence jobs.
+
+package main
+
+// generatePerfClusterUpdatePeriodicJobs generates periodic jobs to update clusters
+// that run performance testing benchmarks
+func generatePerfClusterUpdatePeriodicJobs() {
+	// Generate periodic performance jobs for serving
+	perfClusterUpdatePeriodicJob(
+		"ci-knative-serving-recreate-clusters",
+		recreatePerfClusterPeriodicJobCron,
+		"./test/performance/tools/recreate_clusters.sh",
+		[]string{},
+		"serving",
+		"performance-test",
+	)
+	perfClusterUpdatePeriodicJob(
+		"ci-knative-serving-update-clusters",
+		updatePerfClusterPeriodicJobCron,
+		"./test/performance/tools/update_clusters.sh",
+		[]string{},
+		"serving",
+		"performance-test",
+	)
+
+	// Generate periodic performance jobs for eventing
+	perfClusterUpdatePeriodicJob(
+		"ci-knative-eventing-recreate-clusters",
+		recreatePerfClusterPeriodicJobCron,
+		"./test/performance/performance-tests.sh",
+		[]string{"--recreate-clusters"},
+		"eventing",
+		"eventing-performance-test",
+	)
+	perfClusterUpdatePeriodicJob(
+		"ci-knative-eventing-update-clusters",
+		updatePerfClusterPeriodicJobCron,
+		"./test/performance/performance-tests.sh",
+		[]string{"--update-clusters"},
+		"eventing",
+		"eventing-performance-test",
+	)
+}
+
+// generatePerfClusterReconcilePostsubmitJob generates postsubmit job for the
+// repo to reconcile clusters that run performance testing benchmarks.
+func generatePerfClusterReconcilePostsubmitJob(repo string) {
+	// TODO(chizhg): also add postsubmit jobs for other repos after they set up
+	// performance test infra.
+	if repo == "knative/eventing" {
```

Comment:
This should be a setting in config_knative.yaml.

Reply:
Ditto.